### PR TITLE
Added Contributing.md

### PR DESCRIPTION
**Pull request by rishabh-bansal**

### CODE REVIEW

**Comment by RogerCreasy** [comment]
@rishabh-bansal There is another very similar PR #73 by @fairbraxton
If you don't mind, I will accept his PR, and you can do a PR for the changes. Please respond here if this is OK with you. I want to help as many as possible get their PRs for #Hacktoberfest

**Comment by fairbraxton** [comment]
Yeah, that's fine!

**Comment by RogerCreasy** [comment]
I just merged PR #73 
The file still needs at least:
Section - Pull Request Process
Could you reword this to better match a PHP PR workflow?
Right now, I am the only approver. Perhaps this point could be reworded to include that the PR will be merged after review. Some PR approvals may be delayed for an opportunity for the community to review. (just an idea)

**Comment by RogerCreasy** [comment]
@rishabh-bansal Since this PR is a duplicate of PR #73 I am going to close this one. The requested changes to that PR are still open. You are welcome to work on those, if you want. Please let everyone know by commenting on the issue.
